public/components/common/hooks/use-value-suggestion.ts

```diff
@@ -69,7 +69,7 @@ export const useValueSuggestion = (
       : await data.autocomplete.getValueSuggestions({
           query,
           indexPattern: indexPattern as IIndexPattern,
-          field,
+          field: { ...field, toSpec: (options) => field },
```

Comment:
**question**: Is this change required to work? I am not sure if only another change of this PR, solves the problem.